penalty-box.js

```diff
@@ -37,11 +37,11 @@ app.get('/health', function(req, res) {
 /**
  * POST /rate-limit
  * This will check the application's key to make sure a request is allowed
- * 429 Response will be returned if limit has been reached
- * 200 Response will be returned request is allowed
- * X-Rate-Limit-Limit will be returned with the number of requests allowed per hour
- * X-Rate-Limit-Remaining will be returned with the number of requests remaining
- * X-Rate-Limit-Reset will be returned with the time your rate limit will be reset, in epoch_ms
+ * 200 Response will be returned whether response is allowed or not
```

Comment:
Confusing wording again here, response being allowed is not the right way to describe this